src/foam/nanos/crunch/ui/CapabilityInterceptView.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.ui',
+  name: 'CapabilityInterceptView',
+  extends: 'foam.u2.Element',
+
+  implements: [ 'foam.mlang.Expressions' ],
+
+  requires: [
+    'foam.nanos.crunch.Capability'
+  ],
+
+  imports: [
+    'stack'
+  ],
+
+  properties: [
+    { name: 'data' },
+    { name: 'capabilityOptions', class: 'StringArray' }
+  ],
+
+  css: `
+    ^ {
+      width: 1024px;
+      margin: auto;
+    }
+    ^action-container {
+      display: flex;
+      justify-content: space-between;
+      margin: 8px 0;
+    }
+    ^action-container > div > div > * + * {
+      margin-left: 8px;
+    }
+    ^detail-container {
+      overflow-x: scroll;
+    }
+  `,
+
+  methods: [
+    function initE() {
+      this
+        .addClass(this.myClass())
+
+        .startContext({ data: this })
+          .tag(this.CANCEL, { buttonStyle: 'SECONDARY' })
+        .endContext()
+
+        // Container for the detailview
+        .start()
+          .addClass(this.myClass('detail-container'))
+          // TODO: Replace with Capability selection view appropriate for any user
+          .tag(foam.comics.BrowserView.create({
```

Comment:
It's preferable to add models to the `requires` list and then use `this.<model name>` instead. Eg: `this.BrowserView`. Please do that for BrowserView and SearchMode.